loadtest/user/userentity/actions.go

```diff
@@ -90,4 +90,14 @@ func (ue *UserEntity) ViewChannel(view *model.ChannelView)  (*model.ChannelViewR
 	}
 	channelViewResponse, resp := ue.client.ViewChannel(user.Id, view)
 	return channelViewResponse, resp.Error
-}
\ No newline at end of file
+}
+
+func (ue *UserEntity) 	GetChannelUnread(channelId string) (*model.ChannelUnread, error) {
```

Comment:
Can we get rid of the extra tab/space? This reminds me that we'll probably want to add a `gofmt` job to the Makefile.

Reply:
done. ran gofmt on all files